tests/xdc/failover_test.go

```diff
@@ -55,6 +55,8 @@ import (
 	"go.temporal.io/sdk/temporal"
 	sdkworker "go.temporal.io/sdk/worker"
 	"go.temporal.io/sdk/workflow"
+	"google.golang.org/protobuf/types/known/durationpb"
+
```

Comment:
nooo.. don't undo the goimports work

Reply:
oh! @jmbarzee 's script have failed the build check, no?